tests/sdk/test_list.py

```diff
@@ -65,11 +65,10 @@ def test_list_asset_with_filters(client, mocker):
 
 def test_list_asset_with_filters_failure(client, mocker):
     items = [datastore.ALGO]
-    m = mock_requests(mocker, "get", response=[items])
+    mock_requests(mocker, "get", response=[items])
```

Comment:
It looks like it's not necessary at all to mock requests.get here, the ValueError raised by list_algo seems to occur before the call to requests.get is made.

You could probably remove the line entirely.